src/entity/Item.java

```diff
@@ -39,22 +39,22 @@ public Item(final int positionX, final int positionY) {
         this.livingTime.reset();
         item_dx = Math.random() > 0.5 ? 1 : -1;
         item_dy = Math.random() > 0.5 ? 1 : -1;
-        this.setSprite();
+        this.setSprite(Math.random());
         isdetroyed = false;
         this.destructionCooldown = Core.getCooldown(200);
     }
 
+
     /**
      * Set Sprite dot image.
      */
-    public void setSprite() {
-        double type = Math.random();
+    public final void setSprite(double type) {
         if(type < 0.2){ //20%
             this.spriteType = SpriteType.Buff_Item;
-            this.setColor(Color.GREEN);
+            this.setColor(Color.getHSBColor(122,100,99));
```

Comment:
Actually, I also think it would be nice if buff items were green. so I'd have to make a change to the code, but I'm not sure how to do that, so I chose HSB. The HSB color is also green. 😅